local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,39 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(port) {
+  var debuggerURL = 'http://localhost:' + port + '/debugger-ui';
+  console.log('Launching Dev Tools...');
+  opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+    if (err) {
+      console.error('Google Chrome exited with error:', err);
+    }
+  });
+}
+
+function escapePath(path) {
+  return '"' + path + '"'; // " Can escape paths with spaces in OS X, Windows, and *nix
```

Comment:
@frantic What kind of paths do you think will break with this logic? I tested this with paths with spaces and some common symbols used in folders and it worked on both Mac and Windows.

The reason I think it's better to use exec is because to use spawn we'll need to parse the contents of the environment variable to split it into the command, and the array of arguments, and we'll need to implement some kind of parsing/escaping for the contents of the environment variable (In our prototype debugger we are passing 3 different folder paths in this environment variable as: "path_to_node path_to_script path_to_debugger"). So we'll still have an escaping/parsing issue if we go the spawn route. Here at least we can use quotes that seem to work for all the scenarios I've tested so far.